hwy/ops/rvv-inl.h

```diff
@@ -2862,6 +2866,40 @@ HWY_API VI TableLookupBytesOr0(const VT vt, const VI idx) {
   return BitCast(di, IfThenZeroElse(detail::LtS(idx8, 0), lookup));
 }
 
+// ------------------------------ TwoTablesLookupLanes
+
+template <class D, HWY_IF_POW2_LE_D(D, 2)>
```

Comment:
I think we will have to special-case 8-bit lanes because their (2-table) indices are likely to exceed 255: even with LMUL=4 that will happen for 512-bit vectors, and I know of actual silicon with 2048 bits.
The 8-bit special case can use MaskedTableLookupLanes16 (or a variant without masking).